controllers/topics.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../models/topics.js");
+
+exports.getTopics = (req, res, next) => {
+  return fetchTopics()
+    .then((topics) => {
+      res.send({ topics });
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
If all you are doing is console.log this, I would remove the .catch as it's not yet needed.